.gitlab/e2e/e2e.yml

```diff
@@ -101,6 +101,10 @@ k8s-e2e-otlp-main:
     E2E_OUTPUT_DIR: $CI_PROJECT_DIR/e2e-output
   script:
     - inv -e new-e2e-tests.run --targets $TARGETS -c ddagent:imagePullRegistry=669783387624.dkr.ecr.us-east-1.amazonaws.com -c ddagent:imagePullUsername=AWS -c ddagent:imagePullPassword=$(aws ecr get-login-password) --junit-tar "junit-${CI_JOB_NAME}.tgz" ${EXTRA_PARAMS}
+  after_script:
+    # Upload generated junit files
+    - export DATADOG_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.datadog_api_key_org2)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What if, instead of exporting the key, you prepend it to each call to `inv junit-upload`?

Reply:
yes, regarding yesterday's incident it's probably preferable not to "store" secret keys. I prefer to have a dedicated task on it that will target params one by one if you agree